client/packages/invoices/src/InboundShipment/DetailView/SidePanel/SidePanel.tsx

```diff
@@ -79,7 +86,7 @@ export const SidePanel: FC = () => {
       <RelatedDocumentsSection />
       <PricingSection />
       {isTransfer && <TransportSection />}
-      <DonorSection />
+      {prefs?.allowTrackingOfStockByDonor && <DonorSection />}
```

Comment:
Feels a bit hidden here... can this be moved to be in the additional info section? 👀